src/java/org/apache/cassandra/nodes/Nodes.java

```diff
@@ -492,10 +508,17 @@ private synchronized void snapshot(Path snapshotPath) throws IOException
 
         private void write(OutputStream output) throws IOException
         {
+            lastLocalSave = System.nanoTime();
```

Comment:
Careful if you do this in C*5.0 as system clock calls are forbidden in favor of a simulator friendly wrapper: https://github.com/apache/cassandra/blob/cassandra-5.0/.build/checkstyle.xml#L66

Reply:
I am afraid there isn't any sensible alternative in CC (`4.0.11` based), or am I missing it?

Reply:
Nope. Only raising awareness for future probable C* 5.0 codebase coming our way.